### ISSUE TITLE
Add account import thru polkadot.js extension

### ISSUE THREAD

**Issue by Nexus2k** [subledgr/subledgr.com#5]
Since polkadot.js extension supports accounts from many different sources (e.g. hard wallets from ledger, parity signer, etc.) it would be great to be able to import addresses from the extension instead of manually copy pasting.

**Comment by dcolley** [comment]
It could be an option to import from PJS and then create them in the DB.
Due to storage and indexing we need the address in the backend database.

**Comment by Nexus2k** [comment]
Yeah that would be fine. An import screen that shows all accounts respecting the network filter of the extension and then checkboxes for each account where user can choose which to import.
